taxonomy/admin.py

```diff
@@ -27,14 +27,14 @@ class CourseSkillsTitleAdmin(admin.ModelAdmin):
     Administrative view for Course Skills.
     """
 
-    list_display = ('id', 'course_id', 'created', 'modified')
-    search_fields = ('course_id',)
+    list_display = ('id', 'course_key', 'created', 'modified')
+    search_fields = ('course_key',)
 
     def save_model(self, request, obj, form, change):
         """
-        Set `course_key` to the same value as `course_id` before saving the model.
+        Set `course_id` to the same value as `course_key` before saving the model.
         """
-        obj.course_key = obj.course_id
+        obj.course_id = obj.course_key
```

Comment:
Why do we need to save the course_id now that we have discarded it?

Reply:
Complete details are on [this](https://openedx.atlassian.net/browse/ENT-4402?focusedCommentId=538142) comment.

In short, we can not remove the course id yet as doing so causes integrity error when creating new instances of course skills via either django admin or management command. This field will be removed in the next release.